webui/react/src/components/CompareParallelCoordinates.tsx

```diff
@@ -148,80 +147,86 @@ const CompareParallelCoordinates: React.FC<Props> = ({
     selectedHParams,
   ]);
 
+  const extractHyperparams = useCallback(
+    (
+      record: TrialItem | FlatRun,
+      metricsMap: Record<number, number>,
+      hpMap: Record<string, Record<number, Primitive>>,
+    ) => {
+      if (!selectedMetric) return;
+      const idKey = isRun(record) ? record.id : record.experimentId;
```

Comment:
nit: might make it slightly easier to remove experiments-specific code in the future to comment the parts that are dependent on `selectedExperiments`/`trials` vs `selectedRuns` props.